validation/note.js

```diff
@@ -0,0 +1,40 @@
+import isEmpty from "lodash.isempty";
+import { isBlank, isValidModelId } from "./validationUtils";
+import Note from "../models/note";
+
+export const validateNoteUpdate = (noteId, text) => {
```

Comment:
there are 2 similar validation methods. also you passing id which is not using. You can add validation for id here